boskos/cmd/cli/cli_test.go

```diff
@@ -0,0 +1,17 @@
+/*
```

Comment:
Remove ?

Reply:
Oops, need to add the test :man_facepalming: